packages/office-addin-dev-certs/src/verify.ts

```diff
@@ -12,6 +12,8 @@ function getVerifyCommand(): string {
           return `powershell -ExecutionPolicy Bypass scripts\\verify.ps1 '${defaults.certificateName}'`;
        case "darwin": // macOS
           return `security find-certificate -c '${defaults.certificateName}' -p | openssl x509 -checkend 86400 -noout`;
+        case "linux":
+          return `[ -f /usr/local/share/ca-certificates/${defaults.certificateName} ]`;
```

Comment:
This should be [ -f /usr/local/share/ca-certificates/${defaults.caCertificateFileName} ]. Still this doesn't tell if certificate is expired or not

Reply:
Thanks, will fix.

Yeah, it's fairly naive. Off the cuff, we could do something like `openssl verify /usr/local/share/ca-certificates/${defaults.caCertificateFileName}`. Will do two seconds of research to make sure that's legit.

Reply:
I opted to use the same command used to verify the cert on MacOS, which just checks it won't expire within 86400s.